### PR TITLE
New package: cloud-utils-0.33

### PR DESCRIPTION
**Pull request by akhiljalagam**
#### Testing the changes
- I tested the changes in this PR: **YES**

#### New package
- This new package conforms to the [quality requirements](https://github.com/void-linux/void-packages/blob/master/Manual.md#quality-requirements): **YES**

### CODE REVIEW

File: srcpkgs/cloud-utils/template [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+#conf_files=""
+#make_dirs="/var/log/dir 0755 root root"
+hostmakedepends=""
+makedepends=""
+depends=""
+short_desc="This package provides a useful set of utilities for interacting with a cloud"
+maintainer="akhil <akhil.j@outlook.in>"
+license="GPL-3.0-or-later"
+homepage="https://github.com/canonical/cloud-utils/"
+#changelog=""
```
Comment (meator) [inline srcpkgs/cloud-utils/template:21]: Please remove all unnecessary comments and empty variables.
```suggestion
build_style=gnu-makefile
short_desc="This package provides a useful set of utilities for interacting with a cloud"
maintainer="akhil <akhil.j@outlook.in>"
license="GPL-3.0-or-later"
homepage="https://github.com/canonical/cloud-utils/"
```

File: srcpkgs/cloud-utils/template [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+build_style=gnu-makefile
+#configure_args=""
+#make_build_args=""
+#make_install_args=""
+#conf_files=""
+#make_dirs="/var/log/dir 0755 root root"
+hostmakedepends=""
+makedepends=""
+depends=""
+short_desc="This package provides a useful set of utilities for interacting with a cloud"
```
Comment (meator) [inline srcpkgs/cloud-utils/template:17]: `xlint` (from the package `xtools`) reports:
```
template:17: short_desc should be less than 72 chars
```

**Review by meator — CHANGES_REQUESTED** [review]

**Comment by Chocimier** [comment]
- Should specify its dependencies. 
- We do not have cdrkit packaged.
- Alpine packages for all archs.

File: srcpkgs/cloud-utils/template (hunk 2 above)
Comment (akhiljalagam) [reply srcpkgs/cloud-utils/template]: resolved it.

**Comment by meator** [comment]
There has to be one commit per package. Please squash the three commits into one. https://github.com/void-linux/void-packages/blob/master/CONTRIBUTING.md#review

**Comment by akhiljalagam** [comment]
I messed up this PR. will create new one.

**Comment by akhiljalagam** [comment]
recreated this PR
https://github.com/void-linux/void-packages/pull/37968

**Pull request closed without merging**